src/Mod/Sketcher/Gui/CommandSketcherBSpline.cpp

```diff
@@ -1055,6 +1055,8 @@ void CmdSketcherInsertKnot::activated(int iMsg)
                                 .arg(QString::fromUtf8(multList.str().c_str())));
         // use an appropriate middle value from `knots`
         knotDialog.setDoubleValue(0.5 * (knots.front() + knots.back()));
+        // set step size dependent on the knot range
+        knotDialog.setDoubleStep(0.001 * (knots.back() - knots.front()));
```

Comment:
Is it really useful to have 1000 steps? I think 100 steps for the spin box is more practical. (you can enter very value directly.)

Reply:
I don't have a preference either way but I'll wait for other people for input on this. @abdullahtahiriyo, opinion?